public/app/plugins/panel/canvas/editor/element/QueryParamsEditor.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FormEvent, useState } from 'react';
```

Comment:
It seems like this PR may be blocked by figuring out / merging in the [support adding parameters PR](https://github.com/grafana/grafana/issues/74220)